pkg/backporter/httpcache_test.go

```diff
@@ -0,0 +1,180 @@
+package backporter
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"net/http/httputil"
+	"reflect"
+	"testing"
+)
+
+//bugzillaCache tests
+func TestBugzillaCacheSet(t *testing.T) {
+	testcases := []struct {
+		name         string
+		cache        map[string][]byte
+		key          string
+		value        []byte
+		postSetCache map[string][]byte
+	}{
+		{
+			name:         "Entry is already present",
+			cache:        map[string][]byte{"key": []byte("Test string")},
+			key:          "key",
+			value:        []byte("New data"),
+			postSetCache: map[string][]byte{"key": []byte("New data")},
+		},
+		{
+			name:         "New entry",
+			cache:        map[string][]byte{},
+			key:          "key",
+			value:        []byte("New data"),
+			postSetCache: map[string][]byte{"key": []byte("New data")},
+		},
+	}
+	for _, tc := range testcases {
+		t.Run(tc.name, func(t *testing.T) {
+			bc := newBugzillaCache()
+			bc.cache = tc.cache
+			bc.set(tc.key, tc.value)
+			if !reflect.DeepEqual(tc.postSetCache, bc.cache) {
+				t.Errorf("cached entry mismatch")
+			}
+
+		})
+	}
+
+}
+func TestBugzillaCacheGet(t *testing.T) {
```

Comment:
Not sure if this test is needed, everything we test here is also tested in `TestRoundTrip` already I think?

Reply:
It's not exactly the same test, and as you mentioned in a previous comment having a separate test for the Get function keeps with the spirit of unit testing I guess. I can remove them if you strongly feel that its unnecessary